### DISCUSSION TITLE
Confirm to Play

### DISCUSSION THREAD

**Discussion by iBanks3** [arichornlover/uYouEnhanced#335] [Ideas]
Possible to add an option that pops up to confirm the play of a tapped video? I so often tap another video while watching one on accident and the tapped video begins playing.

**Comment by ego-lay-atman-bay** [comment]
You can turn off autoplay video in uYou settings, which does not autoplay a video whenever you tap on it, it instead pauses the video before it plays.

**Reply by iBanks3** [reply]
True. I was hoping for a way to not switch from the current video at all unless confirmed.

**Reply by ego-lay-atman-bay** [reply]
Yeah, that would also be nice
